code/modules/mob/living/carbon/human/species_types/supersoldier.dm

```diff
@@ -16,4 +16,24 @@
 		if(light_amount > 0) //if there's any light, heal
 			H.heal_overall_damage(1,1, 0, BODYPART_ORGANIC)
 			H.adjustToxLoss(-1)
-			H.adjustOxyLoss(-1)
```

Comment:
why are you nerfing the original supersoldier

Reply:
did I accidentally delete that?

Reply:
No, you didn't